### PR TITLE
Updating ChatDisabled and adding new icons

### PR DESCRIPTION
**Pull request by ZilvinasMaciulis**

### CODE REVIEW

**Review by TalZa1810 — CHANGES_REQUESTED** [review]
Hi @ZilvinasMaciulis 
I've added few comments.
Also, if the icons [here](https://github.com/wix/wix-ui/pull/1973) related to this PR, please update this PR instead of creating a new one.

File: packages/wix-ui-icons-common/src/general/raw/Microphone.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
```
[the remark this answers is not in the document]
Comment (roibWix) [reply packages/wix-ui-icons-common/src/general/raw/Microphone.svg:1]: added in another commit

**Comment by gendeld** [comment]
packages/wix-ui-icons-common/src/general/raw/ViedeoCameraDisabled.svg
Wrong icon name

**Review by gendeld — CHANGES_REQUESTED** [review]
packages/wix-ui-icons-common/src/general/raw/ViedeoCameraDisabled.svg
Wrong icon name

**Review by gendeld — APPROVED** [review]

**Pull request merged**